releasenotes/notes/vf2layout-4cea88087c355769.yaml

```diff
@@ -0,0 +1,7 @@
+---
+features:
+  - |
+    A new pass :class:`qiskit.transpiler.passes.layout.vf2_layout.VF2Layout` is
+    introduced. This pass models the layout allocation problem as a subgraph
+    isomorphism problem. For this, it uses VF2++, a state of the art heurisitic to find
```

Comment:
This is only true if you set `id_order=False`. The vf2++ heuristic matching order is not used when you call `vf2_mapping()` with `id_order=True`.

Reply:
```suggestion
    isomorphism problem. For this, it uses VF2, a state of the art heuristic to find
```